ecosystem/sf-indexer/aptos-sf-indexer/build.rs

```diff
@@ -0,0 +1,10 @@
+// Copyright (c) Aptos
+// SPDX-License-Identifier: Apache-2.0
+
+fn main() {
+    println!("cargo:rerun-if-changed=proto");
+    tonic_build::configure()
+        .out_dir("src/proto")
+        .compile(&["proto/substreams.proto"], &["proto"])
+        .expect("Failed to compile Substreams proto(s)");
+}
```

Comment:
can we use buf instead of tonic pls?

Reply:
We are autogenerating a tonic server here. Can buf do the same?

Reply:
yes: https://crates.io/crates/protoc-gen-tonic